frontend/src/components/ProductList.tsx

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useState } from "react";
+
+const API_URL = "https://58qrqx1y53.execute-api.us-east-1.amazonaws.com/prod/products";
```

Comment:
I think that this URL should be : https://3ooyz8ntq1.execute-api.us-east-1.amazonaws.com/prod/products